faststream/specification/asyncapi/v3_0_0/schema/__init__.py

```diff
@@ -1,3 +1,5 @@
+from faststream.specification.asyncapi.v2_6_0.schema import ServerVariable
```

Comment:
We shouldn't reexport `v2_6_0` classes here, please, reexport them in the final file like here: https://github.com/airtai/faststream/blob/0.6.0/faststream/specification/asyncapi/v3_0_0/schema/utils.py